### DISCUSSION TITLE
"全部话题"窗口设置

### DISCUSSION THREAD

**Discussion by JBJSW** [lobehub/lobehub#1062] [General]
### 🥰 需求描述

1. 为了实现类似于GPTs的定制功能，需要使用prompt设置不同的助手机器人。

2. 助手机器人如图所示
![11](https://github.com/lobehub/lobe-chat/assets/113880070/4ef64605-73cb-4ad1-ad59-9ad38c1ee536)

3. 平常的使用流程需要在多个助手之间频繁切换。如①->②->③->④

4. 目前的UI，需要先定位到机器人，再定位到对应的“话题”，非常低效

### 🧐 解决方案

在“话题”窗口（即窗口⑤）中，设置切换按钮，以实现在**A**与**B**两个模式之间切换：

**模式A：** UI当前模式，即只显示对应机器人下的话题历史
**模式B：** 综合显示全部话题，点击对应话题即跳转至对应机器人下的聊天界面

模式B有些类似于LibreChat的UI设计理念

[a comment by an automated account is not shown]

**Comment by canisminor1990** [comment]
设计思路不太一样，详情可见 https://github.com/lobehub/lobe-chat/wiki/Usage-Topics.zh-CN

同时后续会增加跨助手 `@` 调用功能，以增加灵活度，refs: https://github.com/lobehub/lobe-chat/discussions/1043

**Comment by arvinxx** [comment]
@JBJSW 我们后续也有计划做 #1043 ，应该也会方便很多
